go-sdk/generate-docker.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+
+if ! which docker 2> /dev/null
+then
+  echo "Error: 'docker' executable not found"
+  exit 1
+fi
+
+read -r -d '' SCRIPT <<- END
+apt-get update &&
+apt-get -y install unzip &&
+apt-get -y install libicu-dev &&
+chmod u+rw /registry &&
```

Comment:
This reminds me that the generated files are going to belong to the user used in the container(`root`?) ...

Reply:
I've tested this and the files belong to the user who is running the script (from the outside).

Reply:
For whatever reason this definitely not true for me.  Generated files are created as root for me.  Might depend on how you have docker configured, or if you're using docker vs. podman.